packages/mobile/src/web3/dataEncryptionKey.ts

```diff
@@ -127,6 +128,7 @@ export function* registerAccountDek(walletAddress: string) {
   try {
     const isAlreadyRegistered = yield select(isDekRegisteredSelector)
     if (isAlreadyRegistered) {
+      yield call(uploadProfileInfo)
```

Comment:
This is only called when you try to send to someone. How would you feel about running something once when the app opens (maybe in the `rootSaga` in `src/redux/sagas`) that calls this method and checks for sent transactions on the feed so as to give them read access? If you don't think this PR is the right place maybe we could create an issue to do that?

Reply:
The issue is that this saga needs to be called only after the DEK has been registered on-chain. It seems like `registerAccountDEK` is only called after a successful verification or when sending/inviting someone. Should we consider calling `registerAccountDEK` earlier too?

Reply:
I think that would be great, yes.

For most users though, noone will be able to see their name/picture until they're given access, and we only give access when sending, so maybe it's not _that_ important 🤔

Reply:
Hm I'm a little confused, this is only called to make sure that existing user accounts upload their profiles after this update is pushed out. Profile access is given via `giveProfileAccess`, which is called whenever they send a transaction

Reply:
Yeah, I'm sorry, I confused myself as well.

I think the ideal scenario would be to check on app open just once (similar to how this `isAlreadyRegistered` works) and register the DEK and give access to all people we've sent funds to in the past so that they can see our name and picture. Just one thing to take into account is that I'm not sure if we're caching the feed, so we should make sure the feed is loaded when we do that.

If you feel that's too much extra work for this PR I think it's fine to create an issue for that and do it in another PR, but it would be fine to included it here as well.